nautobot/docs/plugins/development.md

```diff
@@ -170,19 +201,25 @@ class Animal(models.Model):
         return self.name
 ```
 
-Once you have defined the model(s) for your plugin, you'll need to create the database schema migrations. A migration file is essentially a set of instructions for manipulating the PostgreSQL database to support your new model, or to alter existing models. Creating migrations can usually be done automatically using Django's `makemigrations` management command.
+Once you have defined the model(s) for your plugin, you'll need to create the database schema migrations. A migration file is essentially a set of instructions for manipulating the database to support your new model, or to alter existing models.  
+
+Creating migrations can be done automatically using the `nautobot-server makemigrations <plugin_name>` management command, where `<plugin_name>` is the name of the Python package for your plugin (e.g. `animal_sounds`):
+
+```no-highlight
+$ nautobot-server makemigrations animal_sounds
```

Comment:
Do we have some inconsistencies on the plugin name/slug in this document? In the below example output it's `nautobot_animal_sounds` but here and above you're using `animal_sounds`.